h/presenters.py

```diff
@@ -19,10 +19,12 @@ def _format_document_link(href, title, link_text, hostname):
         double-escaped.
 
     """
-    if hostname and hostname in link_text:
+    if href and hostname and hostname in link_text:
```

Comment:
It wasn't initially clear to me reading this that `hostname` could instead be the filename of the document. Worth adding a comment in the function's docstring?